OpenDreamShared/Network/Messages/MsgAllAppearances.cs

```diff
@@ -217,6 +221,9 @@ public override void WriteToBuffer(NetOutgoingMessage buffer, IRobustSerializer
 
             lastId = pair.Key;
 
+            buffer.Write((byte)Property.Name);
+            buffer.Write(appearance.Name);
```

Comment:
You could avoid writing this property if it's equal to `IconAppearance.Default.Name`